test/platform_test/platform_test.go

```diff
@@ -56,6 +57,7 @@ var _ = test.DescribeTestsFor(test.TestCase{
 			"zid": "tenantID",
 		},
 	},
+	SupportsAsyncOperations:                true,
```

Comment:
You can remove this and from visibilities also as we decided that only brokers will be able to processed asynchronously.

Reply:
Is this because we don't have a mechanism to return the platform credentials other way?